### PR TITLE
docs(PPDSC-2553): update accordion docs

### PR DESCRIPTION
**Pull request by messmike**
PPDSC-2553

**What**

Copy updates to the Accordion component page
Copy updates to the Menu component page
Copy updates to the Tabs component page

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2553-navigation-content-updates
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2553-navigation-content-updates/storybook

**Comment by messmike** [comment]
Please help a poor designer fix his first (and maybe last) branch @newskit-engineering!

I have this build error in circleci and for the life of me I can't get this to work:

https://app.circleci.com/pipelines/github/newscorp-ghfb/newskit/4344/workflows/f4d9a7c3-7be7-4ad1-a521-0196e85e31d6/jobs/87529/parallel-runs/0/steps/0-103

Please and thank you in advance.

File: site/pages/components/tabs.tsx [hunk 1]
```diff
@@ -12,12 +12,14 @@ import {
   logicalPaddingOverrideProps,
 } from '../../components/component-api/common-logical-props';
 
+type NewType = any;
```
Comment (Xin00163) [inline site/pages/components/tabs.tsx:15]: Could you please delete this? Once those two are deleted, it should be OK

Comment (messmike) [reply site/pages/components/tabs.tsx:15]: Will do, thank you Xin.

File: site/pages/components/tabs.tsx [hunk 2]
```diff
@@ -862,23 +860,24 @@ const TabsComponent = (layoutProps: LayoutProps) => (
...
-            </> // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          ) as any,
-
+              <InlineMessage role="region" aria-label="note">
+                The default alignment depends on the orientation. When tabs are
+                vertical it’s <Mono>Start</Mono> and when tabs are horizontal
+                it’s <Mono>Center</Mono>.
+              </InlineMessage>
+            </>
+          ) as NewType,
```
[the remark this answers is not in the document]
Comment (messmike) [reply site/pages/components/tabs.tsx:880]: Will do, thank you Xin.

**Review by mutebg — APPROVED** [review]

File: site/pages/components/tabs.tsx (hunk 1 above)
Comment (LukeFinch) [reply site/pages/components/tabs.tsx]: Resolved in my latest commit :) https://github.com/newscorp-ghfb/newskit/pull/452/commits/de6fa10a3342d89544c9de7a649f996b7a13e25a

**Review by LukeFinch — APPROVED** [review]
LGTM

**Review by mutebg — APPROVED** [review]

**Pull request merged**